guides/release/routing/controllers.md

```diff
@@ -6,7 +6,7 @@ The model is passed from the Route to the Controller by default using the [`setu
 
 A Controller is usually paired with an individual Route of the same name.
 
-[Routing Query Parameters](../routing/query-params/) should be defined within a controller.
+[Routing Query Parameters](query-params) should be defined within a controller.
```

Comment:
This was already correct- and working from what I can see- @locks or @mansona did something change that would necessitate this as a fix?

Reply:
@MelSumner Confirmed that this **is** a problem on the Octane branch. It isn't an issue on the regular guides, which threw me for a loop too

Reply:
And I was wrong above. @shankarsridhar when I visit the controllers page at https://deploy-preview-931--ember-guides.netlify.com/release/ this change to the link seems to be causing problems. Do you see that on your end?